package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "hof-template-partials",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "transpile": "hof-transpiler ./translations/src -s FIXME",
+    "postinstall": "npm run transpile"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
We normally use GPLv3 https://www.gnu.org/licenses/quick-guide-gplv3.html. Is there a reason for using ISC?